washingtonpost/klaxon
# NAPPS-1345: run migrations on remote dev database cluster

The bulk of this PR is documentation for how I ran migrations on the new remote database. Now, we should be able to deploy the app to an ECS cluster with connections to this remote db.

JIRA: [NAPPS-1345](https://arcpublishing.atlassian.net/browse/NAPPS-1345)

Testing steps:
- Read through the instructions for clarity
- Try opening a shell on the remote db and confirm the Klaxon tables are there
- As you read through, keep in mind we're trying to contribute these changes to the existing Klaxon, so things should be written with a broader audience in mind and with the intent to avoid merge conflicts

[NAPPS-1345]: https://arcpublishing.atlassian.net/browse/NAPPS-1345?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
